templates/setenv.sh.j2

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+export JAVA_OPTS="$JAVA_OPTS {{tomcat8_java_opts|join(' ')}}"
```

Comment:
This is kind of a nitpick, so feel free to ignore, but I find jinja templates way easier to read with spaces after the opening delimiter and before the closing delimiter like this:
`{{ tomcat8_java_opts|join(' ') }}`

Reply:
done